doc/source/_static/custom.css

```diff
@@ -39,6 +39,143 @@ html[data-theme="dark"] {
   border: none;
 }
 
+/* -------------------- HOMEPAGE -------------------- */
+#key-features,
+#sponsors,
+#using-arviz,
+#supporting-arviz {
+  padding: 3rem 0 7rem;
+}
+
+#key-features h1,
+#sponsors h1,
+#using-arviz h1,
+#supporting-arviz h1 {
+  font-weight: bold;
+  font-size: 48px;
+  text-align: center;
+  margin-bottom: 4rem;
+}
+
+#hero .headerlink,
+#key-features .headerlink,
+#sponsors .headerlink,
+#using-arviz .headerlink,
+#supporting-arviz .headerlink {
+  visibility: hidden;
+  position: absolute;
+}
+
+/* Section hero */
+#hero {
+  display: flex;
+  flex-direction: row;
+  padding: 3rem 0 7rem;
+  min-height: min(calc(80vh), 1100px); /* Make hero fill up most of the page on load */
+}
+#hero-left {
+  max-width: 476px;
+  width: 40%;
+  margin: auto 0;
+}
+#hero-right {
+  min-width: 476px;
+  width: 60%;
+  margin: auto 0;
+}
+
+.homepage-button-container {
+  margin-bottom: 1rem;
+  display: flex;
+  flex-direction: column;
+}
+.homepage-button-container-row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.homepage-button-container a {
+  transition: 0.1s;
+}
+.homepage-button {
+  min-width: 142px;
+  padding: 0.5em 2em;
+  border: 1px solid var(--pst-color-primary);
+  border-radius: 4px;
+  margin: 1em 0.5em 0.5em 0;
+}
+.primary-button {
+  background-color: var(--pst-color-primary);
+  color: var(--pst-color-background);
+}
+.secondary-button {
+  background-color: var(--pst-color-background);
+  color: var(--pst-color-primary);
+}
+.homepage-button:hover {
+  text-decoration: none;
+  background-color: var(--pst-color-secondary);
+  color: var(--pst-color-background);
+  border: 1px solid var(--pst-color-secondary);
+}
+.homepage-button-link {
+  text-decoration: underline;
+}
+
+/* Key Features */
+#key-features .sd-card-body {
+  display: flex;
+}
+#key-features .sd-card img {
+  width: 140px;
+  height: 140px;
+}
+#key-features .sd-card-body .key-features-text {
+  min-width: 70%;
+  padding-top: 20px;
+}
+
+/* Section Support ArviZ */
+#using-arviz h3,
+#supporting-arviz h3 {
+  font-weight: bold;
+  text-align: center;
+}
+#using-arviz .sd-col p,
+#supporting-arviz .sd-col p {
+  text-align: center;
+}
+
+/* Sponsors */
+img {
+  max-width: 200px;
+}
```

Comment:
Just realized this is being applied to _all_ images throughout the website. For example: https://arviz--2119.org.readthedocs.build/en/2119/examples/plot_forest_mixed.html